### ISSUE TITLE
[Quest][Classic] Protecting the shipment - Doesn't start

### ISSUE THREAD

**Issue by nemmjuuk** [dalaranwow/dalaran-wow#5196]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Talk to quest giver to start quest. NPC just stands there and nothing more happens. 

**Expected behaviour**: Tell us what should happen instead
NPC should start moving so we can escort him.

**Steps to reproduce the problem**:

1. Take quest
2. Watch nothing happening 
3. 

**Include proofs for this behaviour**
Verified with alts of different classes and levels. 
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Quest ID 309
https://db.vanillagaming.org/?quest=309

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by nemmjuuk** [comment]
Quest can be completed, but it would be nice if there was some actual ambush etc..

**Comment by TeridanDW** [comment]
Hello and thank you for the report @nemmjuuk,
Issue confirmed on PTR, escort event is completely missing.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
